src/Board/GomukuBoard.cpp

```diff
@@ -5,33 +5,54 @@
 #include "GomukuBoard.hpp"
 
 void GomukuBoard::set(int x, int y, bool isPlayer) {
-    int invertedIndex = (BOARD_SIZE - 1 - y) * BOARD_SIZE + (BOARD_SIZE - 1 - x);
     if (isPlayer) {
-        player.set(invertedIndex);
+        player.set(x, y);
     } else {
-        opponent.set(invertedIndex);
+        opponent.set(x, y);
     }
+    if (minX > x) minX = x;
+    if (maxX < x) maxX = x;
+    if (minY > y) minY = y;
+    if (maxY < y) maxY = y;
```

Comment:
les attributs tu peux mettre un `_` devant ? pour qu'on sache ça sort pas du trou du cul du monde ?

```suggestion
    if (_minX > x) _minX = x;
    if (_maxX < x) _maxX = x;
    if (_minY > y) _minY = y;
    if (_maxY < y) _maxY = y;
```